install.sh

```diff
@@ -322,22 +324,22 @@ while true; do
     -b|--branch)
         tensile_tag=${2}
         shift 2 ;;
-    -t|--test_local_path)
+    -t|--test-local-path)
         tensile_test_local_path=${2}
         shift 2 ;;
-    -n|--no_tensile)
+    -n|--no-tensile)
         build_tensile=false
         shift ;;
     -s|--tensile-host)
         build_tensile_host=true
         shift ;;
-    --build_dir)
+    --build-dir)
```

Comment:
why are you changing the spelling? If there was no request, we risk breaking scripts.